### PR TITLE
add shiplift docker client to remote api client list

### PR DESCRIPTION
**Pull request by softprops**
I'd like to add a Rust client I've been working on to the list of remote api clients.

### CODE REVIEW

File: docs/reference/api/remote_api_client_libraries.md [hunk 1]
```diff
@@ -203,6 +203,12 @@ will add the libraries here.
     </tr>
     <tr>
       <td>Rust</td>
+      <td>shiplift</td>
+      <td><a class="reference external" href="https://github.com/softprops/shiplift>https://github.com/softprops/shiplift</a></td>
```
Comment (thaJeztah) [inline docs/reference/api/remote_api_client_libraries.md]: You forgot a closing quote here (after `href="https://github.com/softprops/shiplift`)

**Comment by thaJeztah** [comment]
Thanks @softprops, can you fix the missing quote, amend your commit, and do a force-push? That should automatically update the PR

**Comment by softprops** [comment]
pushed.

**Comment by thaJeztah** [comment]
@softprops can you squash your commits, so that there's a single commit in the PR?

**Comment by softprops** [comment]
whoops. sorry about that. `squashed` into one commit

**Comment by thaJeztah** [comment]
Oh, boy, apologies, I overlooked this one; can you swap it with `docker-rust`, so that they are in alphabetical order? Really sorry I missed that before :blush:

**Comment by softprops** [comment]
flipped and resquashed

**Comment by thaJeztah** [comment]
Awesome!

LGTM

**Comment by vdemeester** [comment]
LGTM 🐲

**Comment by softprops** [comment]
thanks @thaJeztah !

**Pull request merged**